frontend/src/components/BoardCreationForm/BoardCreationForm.jsx

```diff
@@ -52,14 +52,16 @@ const BoardCreationForm = () => {
               body: JSON.stringify(newBoard),
             };
             const postBoardUrl = `${REST_API_URL}/boards`;
-            return await fetch(postBoardUrl, requestOptions)
-              .then((response) => {
-                return response.json();
-              })
-              .then(window.alert('Board added correctly!'))
-              .then((res) => {
-                navigate(`/board/${res.returnedData.id}`);
-              });
+            return await fetch(postBoardUrl, requestOptions).then(async (response) => {
+              if (response.ok) {
+                await response.json();
+                window.alert('Board added correctly!');
```

Comment:
Tu bym nie używał alertu. Moim zdaniem dobrą informacją potwierdzającą, że się udało jest to, że nas przenosi na stronę tego boarda. Gdyby się nie udało to by nas nie przeniosło.

Reply:
Przenosi na stronę w kolejnym kroku. Wydaje mi się, że potrzeba jasnej informacji dla użytkownika, że board został dodany poprawnie, lub nie (i dlaczego).

Reply:
Jak coś to informuję, że odczytane — ale to już od Was zależy :))